### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by shannonbux**
Removed stuff about .eslintrc.jason per our conversation in Berkeley. Also capitalized all instances of the word "Markdown" (hope that's correct!). 

Wondering about the next few tutorials, since this one mentions "you're halfway through!" and implies there are more tutorials after Part Four. Should we create stubs for those next tutorials?

### CODE REVIEW

File: docs/tutorial/part-four/index.md [hunk 1]
```diff
@@ -563,11 +552,11 @@ _about_ files but what if you want to query data _inside_ files?
 To make this possible, Gatsby supports transformer plugins which take raw
 content from source plugins and _transform_ it into something more usable.
 
-For example, markdown files. Markdown is nice to write in but when you build a
-page with it, you need the markdown to be HTML.
+For example, Markdown files. Markdown is nice to write in but when you build a
```
Comment (KyleAMathews) [inline docs/tutorial/part-four/index.md]: "markdown" should only be capitalized when used as a noun? Maybe? So when an adjective "markdown files" it's not capitalized?

Comment (KyleAMathews) [reply docs/tutorial/part-four/index.md]: Ah never mind! Proper adjectives are capitalized http://englishplus.com/grammar/00000048.htm

TIL

**Comment by KyleAMathews** [comment]
Let's not create stubs... I just need to write the last tutorial 😅 Also by conceptual weight, part 4 is as heavy as the first three so it kinda checks out still.

**Comment by KyleAMathews** [comment]
Thanks!

**Comment by shannonbux** [comment]
k cool, sounds good! Yeah, part 4 does seem like a second half lol.

**Pull request merged**